projects/aca-content/src/lib/ui/theme.scss

```diff
@@ -271,6 +267,14 @@ mat-snack-bar-container {
   border-bottom: 1px solid var(--adf-theme-foreground-text-color-014);
 }
 
+.mat-mdc-tab-list {
+  .mat-mdc-tab-labels {
+    .mdc-tab--active {
+      color: #212121;
```

Comment:
I think we can use var(`$selected-text-color`) here

Reply:
thanks!